### DISCUSSION TITLE
Request to have total in Legend

### DISCUSSION THREAD

**Discussion by userlerueda** [grafana/grafana#39594] [Dashboards, Panels & Visualizations]
It would be nice to have an option to display the Total in panel that uses pie-chart.

In the following example:
<img width="458" alt="Screen Shot 2021-09-23 at 10 46 43 AM" src="https://user-images.githubusercontent.com/5360349/134540296-abb2f861-8a21-4194-b61f-d15216d08c7d.png">

In the legend to have a Total with 1502 under the `Yes` and `No`.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82977. Feel free to continue the discussion around this there. Thank you!
